examples/http-client-server/client/client.bal

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/io;
+import ballerina/lang.'int as ints;
+import ballerina/regex;
+import ballerina/tcp;
+
+type Request record {
+    map<string> headers;
+    string body;
+    string HttpVersion = HTTP_1_1;
+    string Method = POST;
+    string path;
+};
+
+enum Method {
```

Comment:
Let's keep the `enums` together.

Reply:
Fixed all the comments https://github.com/ballerina-platform/module-ballerina-tcp/pull/379/commits/df79651d3030fce4bb9c405de1257c506c417440